### PR TITLE
Gidk 3ng8jj com branch prod

### PR DESCRIPTION
**Pull request by shalinee-walkover**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
https://app.clickup.com/t/3ng8jj

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Other information**:

### CODE REVIEW

File: apps/web-giddh/src/app/shared/primary-sidebar/company-branch/company-branch.component.ts [hunk 1]
```diff
@@ -0,0 +1,429 @@
...
+        this.destroyed$.complete();
+    }
+
+    /**
+     * Company filtering method
+     *
+     * @param {*} ev Modal change event
+     * @memberof CompanyBranchComponent
+     */
+    public filterCompanyList(event: any): void {
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/shared/primary-sidebar/company-branch/company-branch.component.ts:161]: Please check this method is also present in `primary-sidebar.component.ts`, do remove it from there if unused.

File: apps/web-giddh/src/app/shared/primary-sidebar/company-branch/company-branch.component.ts [hunk 2]
```diff
@@ -0,0 +1,429 @@
...
+    }
+
+    /**
+     * Change company callback method
+     *
+     * @param {string} selectedCompanyUniqueName Selected company unique name
+     * @param {boolean} [fetchLastState] True, if last state of the company needs to be fetched
+     * @memberof CompanyBranchComponent
+     */
+    public changeCompany(selectedCompanyUniqueName: string, selectBranchUniqueName: string, fetchLastState?: boolean) {
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/shared/primary-sidebar/company-branch/company-branch.component.ts:181]: Please check this method is also present in `primary-sidebar.component.ts`, do remove it from there if unused.

File: apps/web-giddh/src/app/shared/primary-sidebar/company-branch/company-branch.component.ts [hunk 3]
```diff
@@ -0,0 +1,429 @@
...
+
+    /**
+    * Sets the organization details
+    *
+    * @private
+    * @param {OrganizationType} type Type of the organization
+    * @param {OrganizationDetails} branchDetails Branch details of an organization
+    * @memberof CompanyBranchComponent
+    */
+    private setOrganizationDetails(type: OrganizationType, branchDetails: OrganizationDetails): void {
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/shared/primary-sidebar/company-branch/company-branch.component.ts:206]: Please check this method is also present in `primary-sidebar.component.ts`, do remove it from there if unused.

File: apps/web-giddh/src/app/shared/primary-sidebar/company-branch/company-branch.component.ts [hunk 4]
```diff
@@ -0,0 +1,429 @@
...
+        this.store.dispatch(this.companyActions.setCompanyBranch(organization));
+    }
+
+    /**
+     * Refreshes the company list
+     *
+     * @param {Event} event
+     * @memberof CompanyBranchComponent
+     */
+    public refreshCompanies(event: Event): void {
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/shared/primary-sidebar/company-branch/company-branch.component.ts:221]: Please check this method is also present in `primary-sidebar.component.ts`, do remove it from there if unused.

File: apps/web-giddh/src/app/shared/primary-sidebar/company-branch/company-branch.component.ts [hunk 5]
```diff
@@ -0,0 +1,429 @@
...
+    public openModalCreateNewCompany(): void {
+        this.createNewCompany.emit();
+    }
+
+    /**
+     * Logs out the user
+     *
+     * @memberof CompanyBranchComponent
+     */
+    public logout(): void {
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/shared/primary-sidebar/company-branch/company-branch.component.ts:242]: Please check this method is also present in `primary-sidebar.component.ts`, do remove it from there if unused.

File: apps/web-giddh/src/app/shared/primary-sidebar/company-branch/company-branch.component.ts [hunk 6]
```diff
@@ -0,0 +1,429 @@
...
+        }
+    }
+
+    /**
+     * This will filter branches
+     *
+     * @param {*} event
+     * @memberof CompanyBranchComponent
+     */
+    public filterBranchList(event: any): void {
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/shared/primary-sidebar/company-branch/company-branch.component.ts:367]: Please check a similar method `filterBranch` is also present in `primary-sidebar.component.ts`, do remove it from there if unused.

**Review by prateek-giddh — CHANGES_REQUESTED** [review]

File: apps/web-giddh/src/assets/css/style-2.scss [hunk 7]
```diff
@@ -10092,4 +10169,7 @@ body.prevent-body-scroll {
 }
 .giddh-no-permissions {
     display: none !important;
-}
\ No newline at end of file
+}
+.giddh-no-permissions-error-message {
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/assets/css/style-2.scss:10173]: need to remove this

**Review by ravinderwalkover — CHANGES_REQUESTED** [review]

**Review by prateek-giddh — APPROVED** [review]

**Review by ravinderwalkover — APPROVED** [review]

**Review by ravinderwalkover — APPROVED** [review]

**Pull request merged**